### ISSUE TITLE
Transition to igraph PyPI package instead of python-igraph package

### ISSUE THREAD

**Issue by vtraag** [igraph/python-igraph#412]
At the moment, the Python interface of `igraph` is called `python-igraph`, also on PyPI. However, once installed, you use `import igraph` in order to load the Python interface of `igraph`. This leads many people to be able to simply use `pip install igraph` in order to install the Python interface of `igraph`. However, this does not work, since the Python igraph is also called `python-igraph` on PyPI, and should hence be installed using `pip install python-igraph`. In fact, there is another project called [`igraph` on PyPI](https://pypi.org/project/igraph/), which provides an [embeddable webGL graph visualization library](https://github.com/patrickfuller/jgraph). This other `igraph` project already started using the alternative name `jgraph` in order to avoid name collision with this `igraph` project. This has lead to quite some confusion for users, who unknowingly ran `pip install igraph` instead of `pip install python-igraph` and ended up with an unexpected library (see for example [this StackExchange post](https://stackoverflow.com/questions/36200707/error-with-igraph-library-deprecated-library)).

It would therefore be ideal to transition to start using the `igraph` name on PyPI, so that people can install the Python interface of `igraph` simply using `pip install igraph`, as most people would expect. We discussed this with the current owner of the `igraph` namespace, @patrickfuller, who graciously granted us the opportunity to start using the `igraph` namespace.

The idea would be to start using the `igraph` name on PyPI from a certain version onwards (e.g. 0.10). Earlier versions would then remain available under `python-igraph`, while newer versions would be available from `igraph`. For a brief transitionary period, the Python interface would be available under both names from PyPI, in order to facilitate the transition. Any versions of the `igraph` package that are listed now on PyPI (versions 0.1.0 to 0.1.11) would simply be deleted. That way, anything pointing to `igraph` would always only be able to get the actual Python interface of `igraph`, not the alternative `jgraph` webGL visualization library.

The name `python-igraph` is expected to continue to be used in other contexts. Foremost, this repository will continue to be named `python-igraph`. Many package managers also provide Python packages, which are typically prefixed with something like `python-` or `py-` and they should continue to do so. Each package manager will have its own standards of how to deal with these, and that should all be fine. However, if there are any points that should be discussed in connection with this, we can do so in this issue.

There is at least one such point, and that is the package on the Anaconda [`conda-forge` channel](https://github.com/conda-forge/python-igraph-feedstock). I will open a separate issue for this in Anaconda to discuss this also with the maintainers of `conda-forge`.

**Comment by jarrodmillman** [comment]
I am happy to see you are taking care of it (it has been a minor annoyance of mine).  Another minor annoyance is that there is also:   https://pypi.org/project/igraph-python/

I don't know what to do, but I just wanted to bring it to your attention.

**Comment by ntamas** [comment]
Thanks, we know about this other package and @vtraag has already tried contacting the author to work something out. We haven't received a response until now. There are formal procedures in PyPI for handling situations like this so if we cannot contact the authors for an extended period of time and we can document our attempts, we can turn to the PyPI maintainers to help sort this out.

**Comment by ntamas** [comment]
Transitioning from `python-igraph` to `igraph` has started; the original author of the `igraph` package has kindly handed over the ownership to us; see the [package description of igraph](https://pypi.org/project/igraph/) on PyPI for more details. The current version of `python-igraph` will also be published as `igraph` in PyPI in early September 2021. The old `python-igraph` package will be updated in parallel to `igraph` until September 1, 2022. `python-igraph` will receive no more updates from September 1, 2022.

**Comment by vtraag** [comment]
One further update, the incorrect `igraph-python` package is now also removed from PyPI.
